On windows: build x86 executable fails with C2440

Trying to build an x86 executable fails

**V version:** 
OS: windows, Microsoft Windows 7 Professional v7601 64-bit
Processor: 4 cpus, 32bit, little endian, Intel(R) Core(TM) i5-2500 CPU @ 3.30GHz
CC version: N/A

vroot: D:\Repositories\eko\v
vexe: D:\Repositories\eko\v\v.exe
vexe mtime: 2020-09-30 13:32:44
is vroot writable: true
V full version: V 0.1.29 76e373a.763ddf7

Git version: git version 2.19.1.windows.1
Git vroot status: 0.1.29-459-gcbb12916 (1 commit(s) behind V master)
.git/config present: true
thirdparty/tcc: N/A

**What did you do?**
v -cc msvc D:\scripts\v\test_exceptions.v

**What did you expect to see?**
no error
 
**What did you see instead?**
error C2440: 'function': cannot convert from 'int (__stdcall *)(ExceptionPointers *)' to 'VectoredExceptionHandler'

--- a/vlib/builtin/builtin_windows.c.v
+++ b/vlib/builtin/builtin_windows.c.v
@@ -224,7 +224,7 @@ fn unhandled_exception_handler(e &ExceptionPointers) int {
 }
 
 fn add_unhandled_exception_handler() {
-	add_vectored_exception_handler(unhandled_exception_handler)
+	add_vectored_exception_handler(VectoredExceptionHandler(voidptr(unhandled_exception_handler)))
 }
 
 fn C.IsDebuggerPresent() bool

